### DISCUSSION TITLE
Hey! Is there any Front-end lover?😄

### DISCUSSION THREAD

**Discussion by FangzhouSu** [uestcer/application#150] [General]
This is a Front-end lover who comes from CS faculty, Grade three(OK I know my English is really a Chinese one😑). But I'm just a beginner of Front-end, will you guys be willing to chat with me and share some great things🥺
If so, maybe you can contact with me through QQ：1773279395~

**Comment by FangzhouSu** [comment]
qq联系的原因主要是我前两天拉了一个23届成电前端校招备战 所以想拉一些朋友一起打卡学习 分享些面试经验 岗位信息之类的~想聊啥idea的 github当然是很好的选择XD（突然使用母语）

**Comment by mrcaidev** [comment]
Glad to be on board! I'm also working on frontend these months (mostly on `React`)！ :)
